File: program81.java
//Pattern printing in java

import java.lang.*;
import java.util.*;

class Pattern
{
    public void Display(int no)
    {
        for(int i=1;i<=no;i++)
        {
            System.out.println("*\t");
        }
    }
}

class program81
{
    public static void main(String arg[])
    {
        int ivalue=0;
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the value");
        ivalue=sobj.nextInt();

        Pattern obj=new Pattern();
        obj.Display(ivalue);
    }
}

File: program81.c
//pattern printing
//ip=5
//op=* * * * *
#include<stdio.h>

void Display(int iNo)
{
    int i=0;
    for(i=1;i<=iNo;i++)
    {
        printf("*\t");
    }

}
int main()
{
    int iValue=0;

    printf("Please enter value:\n");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}